content/a/the-greens.md

```diff
@@ -29,10 +28,6 @@ Thus, wrestlers like [Rafi](@/w/rafi.md) or [Sambor](@/w/sambor.md) who also sta
 
 ### Influence
 
-The Greens, aside from being a friends group, were also one of the factions in [Prime Time Wrestling's](@/o/ptw.md) locker room.
-The other factions were "Originals" (who started in PTW) and "Blues" (ex-[KPW](@/o/kpw.md)).
-Sometimes, internal tensions in the organization were caused by disagreement between these groups, and the management.
-Mainly caused by differences in their approach to quality and wrestling philosophy.
-For example: Justin Joy's harsh training regime.
```

Comment:
For some context about this style: it's not necessary to join them together into a single paragraph - GFM does that automatically until the next blank line. So this particular block renders identically before and after the changes.

To make an explicit line break in the middle of a paragraph, end the line with a backslash `\`

Reply:
Wiem, ale wytłumacz to mojemu OCD napędzanemu przez zboczenie zawodowe. :D